src/rules/no-jasmine-globals.ts

```diff
@@ -109,18 +117,26 @@ export default {
         }
       },
       MemberExpression(node) {
-        if (node.object.name === 'jasmine') {
-          if (node.parent.type === 'AssignmentExpression') {
-            if (node.property.name === 'DEFAULT_TIMEOUT_INTERVAL') {
+        if ('name' in node.object && node.object.name === 'jasmine') {
+          const { parent, property } = node;
+
+          if (parent && parent.type === AST_NODE_TYPES.AssignmentExpression) {
+            if (
+              'name' in property &&
+              property.name === 'DEFAULT_TIMEOUT_INTERVAL'
+            ) {
+              const { right } = parent;
+
               context.report({
-                fix(fixer) {
-                  return [
-                    fixer.replaceText(
-                      node.parent,
-                      `jest.setTimeout(${node.parent.right.value})`,
-                    ),
-                  ];
-                },
+                fix:
+                  right.type !== AST_NODE_TYPES.Literal
```

Comment:
Can you flip it so the `true` case is first in the ternary? Or do a `return` above this if it's of the wrong type